utils/config/tokenrefresh.go

```diff
@@ -153,13 +153,18 @@ func refreshAccessTokenAndWriteToConfig(serverConfiguration *ServerDetails, curr
 	if err != nil {
 		return "", errorutils.CheckErrorf("Refresh access token failed: " + err.Error())
 	}
-	err = writeNewArtifactoryTokens(serverConfiguration, tokenRefreshServerId, newToken.AccessToken, newToken.RefreshToken)
+	err = writeNewTokens(serverConfiguration, tokenRefreshServerId, newToken.AccessToken, newToken.RefreshToken, false)
 	return newToken.AccessToken, err
 }
 
-func writeNewArtifactoryTokens(serverConfiguration *ServerDetails, serverId, accessToken, refreshToken string) error {
+func writeNewTokens(serverConfiguration *ServerDetails, serverId, accessToken, refreshToken string, isArtifactoryTokens bool) error {
```

Comment:
I think that this approach tells the whole story, unlike with a boolean.

```suggestion
func writeNewTokens(serverConfiguration *ServerDetails, serverId, accessToken, refreshToken string, tokenType TokenType) error {
```